### ISSUE TITLE
This doesn't appear to work with python>=3.6 type hints

### ISSUE THREAD

**Issue by saml-dev** [jeshan/botostubs#2]
As of python 3.6, type hints have changed from 
```
client = boto3.client('ssm')  # type: botostubs.SSM
```
to
```
client: botostubs.SSM = boto3.client('ssm')
```
and I can't get it to work in VS code. I never used it in python 3.5 so I'm not sure if the new type hints are the issue or it's just not working with my vscode for some other reason, but I wanted to rule out the type hint format change.

**Comment by jeshan** [comment]
This package has nothing IDE-specific, it's a regular PyPI package like any other.
Here it is working in my Intellij IDEA:
![image](https://user-images.githubusercontent.com/1020565/53032165-bd241a80-3487-11e9-9d10-e8b1162f8d27.png)

Please check if type hints from other packages work. If it doesn't, see if others are experiencing the same issue in the VS Code issue tracker.

Let me know how it goes.

**Comment by jeshan** [comment]
any news @SamLewis0602 ?

**Comment by saml-dev** [comment]
Sorry, I haven't had time to figure this out and likely won't soon. Feel free to close this and I can always reopen it once I've got more info. Your screenshot has convinced me that it's something with my environment, I just haven't had the time to dig in and figure out what it is 😁

**Comment by saml-dev** [comment]
These two images confirm that type hints are working in vscode in general:

![image](https://user-images.githubusercontent.com/15079088/53647548-c9d50b00-3c0b-11e9-8bd9-85c1cbefd7b5.png)
-----------------------
![image](https://user-images.githubusercontent.com/15079088/53647574-d78a9080-3c0b-11e9-9ec7-43b21fd51c24.png)

However, I don't get anything from the botostubs type hint.
![image](https://user-images.githubusercontent.com/15079088/53647739-29331b00-3c0c-11e9-9c55-533f32dbeee2.png)
I have the latest versions of boto* libraries as of today:
```
boto3                    1.9.105           
botocore                 1.12.105          
botostubs                0.4.1.9.105 
```

Any thoughts?

**Comment by saml-dev** [comment]
I just discovered, if I enable Jedi as the Intellisense engine in settings (just search jedi in settings to find it) then the botostubs methods work. So this appears to be an issue with type hints. It also looks like they have an as-of-yet unreleased fix here: https://github.com/Microsoft/python-language-server/pull/546

**Comment by jeshan** [comment]
Thanks for letting us know. I had some trouble with VS Code as well. (I
only installed it to test botostubs)

**Comment by pcgeek86** [comment]
I got this working by disabling the Jedi Intellisense engine, opting for the Microsoft Python Language Server instead. That's accomplished by adding the following to my user settings JSON configuration.

```
    "python.jediEnabled": false
```

Also, I posted the issue with the `botostubs` over on the Jedi project. Hopefully they'll be able to fix it. It's hard to say if it's an issue with Jedi or the Python extension for Visual Studio Code.

https://github.com/davidhalter/jedi/issues/1341
